docs/managing_deis/add_remove_host.rst

```diff
@@ -284,3 +284,29 @@ This can be achieved by making a request to the etcd API. See `remove machines`_
 .. _`remove machines`: https://coreos.com/docs/distributed-configuration/etcd-api/#remove-machines
 .. _`removing monitors`: http://ceph.com/docs/hammer/rados/operations/add-or-rm-mons/#removing-monitors
 .. _`removing OSDs`: http://docs.ceph.com/docs/hammer/rados/operations/add-or-rm-osds/#removing-osds-manual
+
+Graceful Shutdown
+~~~~~~~~~~~~~~~~~
+
+The contrib user-data provides a unit which provides some experimental logic to clean-up a Deis
+nodes ceph and etcd membership before reboot, shutdown or halt events. Currently, the procedure will
+also be triggered if it is manually stopped via systemctl.
```

Comment:
What would happen if you gracefully shut down the node with `sudo shutdown 00:00` and didn't install deis-store-admin? Will this script still be run?